tensorflow/lite/model.cc

```diff
@@ -627,6 +627,12 @@ TfLiteStatus InterpreterBuilder::operator()(
     return kTfLiteError;
   }
 
+  if (num_threads < -1) {
```

Comment:
Isn't this check redundant as it will be checked later via (*interpreter)->SetNumThreads(num_threads) in this function?

Reply:
@multiverse-tf : This is for early reporting, i think it is good to have.

Reply:
Fair enough.